lib/absinthe/blueprint/schema.ex

```diff
@@ -145,7 +145,7 @@ defmodule Absinthe.Blueprint.Schema do
   end
 
   defp build_types([{:values, values} | rest], [enum | stack], buff) do
-    enum = Map.update!(enum, :values, &(values ++ &1))
+    enum = Map.update!(enum, :values, &(List.wrap(values) ++ &1))
```

Comment:
What is the need to wrap these in a list?

Reply:
Ahh, yes this is the sneaky bit. Now that `values` are `wrap_in_unquote`'d it can now be a list of atoms _or_ AST for a function call - so I just wrap everything in a list herer so that it can continue to collect in a list.

Then in the end after this all is injected into `__absinthe_blueprint__` it'll be some form of nested list, and I can call `List.flatten(type_def.values)` to get the final resulting list....